doorstop/core/tests/test_fixtures/001-item-references-utf8-keyword/REQ-UTF8.yml

```diff
@@ -0,0 +1,15 @@
+active: true
+derived: false
+header: ''
+level: 1.5
+links:
+- REQ001: 35ed54323e3054c33ae5545fffdbbbf5
+normative: true
+ref: ''
+references:
+- keyword: français
+  path: files/text.txt
```

Comment:
Should this be `path: files/text-utf8.txt` instead?

Reply:
@JustinW80, good catch! Technically you are right but the thing is that the existing test does not use this path directly. Instead the full path is passed manually to the `ReferenceFinder` and it is like that by design of this and other similar tests unit tests. Instead of fixing this path, I have pushed a change that explicitly marks the `path: ` as irrelevant for this test. This should help a future reader to see the intention behind this mock-based unit test that tests ReferenceFinder in isolation.

In my opinion, it is very important that this kind of functionality would also be tested with integration tests without any mocking involved as I proposed back then here: https://github.com/doorstop-dev/doorstop/pull/431. Since then I was attracted away to some other projects so I didn't really have time to push for bringing in the end-to-end test suite (be it based on LIT/FileCheck like I suggested there or something manual).